datalake/src/main/java/com/sequenceiq/datalake/service/sdx/diagnostics/DiagnosticsService.java

```diff
@@ -30,6 +31,9 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DiagnosticsService.class);
 
+    @Value("${info.app.version:}")
+    private String version;
```

Comment:
where is it used?

Reply:
it.is only used as a default if for some reason the blueprint for the cluster does not exist

Reply:
but I cannot see it referenced anywhere